### DISCUSSION TITLE
Qlab / OSC Override

### DISCUSSION THREAD

**Discussion by hadphild** [cpvalente/ontime#1313] [Q&A]
Is there a way to get QLab to send its timers into ontime and overly or add its timer on the display?  While still counting down the session.

**Comment by alex-Arc** [comment]
Hey @hadphild 
Yes, you can use something like [chataigne](http://benjamin.kuperberg.fr/chataigne) or [companion](https://bitfocus.io/companion) to extract the qlab playback state and then send it to Ontime's [external message](https://docs.getontime.no/features/external-message/) field

or you can specify the messages directly in qlab, but you would have to type i all out manually
